Components/RTShaderSystem/src/OgreShaderProgramManager.cpp

```diff
@@ -322,14 +328,22 @@ GpuProgramPtr ProgramManager::createGpuProgram(Program* shaderProgram,
 
             outFile << source;
             outFile.close();
+            loadStore = "stored to ";
```

Comment:
why do you want to log this? You can assume that if the file exists in the RTShaderCache, it will be read on the next run.

If you drop the load/store/cache info, you could just log at the start of this function.

Reply:
As the cached files persist across application invocations, and the cached file content could have been edited (as I did when I was experimenting), it is important to know when the filesystem version is used and when the shader is freshly generated.

The time between application invocations could be large, so it is good to be reminded when Ogre is reading a shader from disk.

Reply:
"cache" is a bit of a misnomer here as it does not save any computations. One would not enable this, unless you want to look at the generated code. So in hindsight using `cachePath` for extended logging should be fine.
This will also immediately tell you whether, "caching" is enabled.